in-toto-spec.md

```diff
@@ -529,106 +538,70 @@ To provide descriptive examples, we will adopt "canonical JSON," as described
 in
 [http://wiki.laptop.org/go/Canonical\_JSON](http://wiki.laptop.org/go/Canonical_JSON),
 as the data format. However, applications that desire to implement in-toto are
-not required to use JSON.  Discussion about the intended data format for
-in-toto can be found in the in-toto website.
-
-### 4.2 File formats: general principles
+not required to use JSON.
 
-All signed files (i.e., link and layout files) have the format:
+With the acceptance and adoption of ITE-5, in-toto is agnostic to the signature
+envelope used for metadata. As such, the specification does not describe the
+previously used signature envelope. However, several examples still make use of
+this signature envelope and its specification can be found in the appendix to
+ITE-5 for reference.
 
-```json
-{
-  "signed" : "<ROLE>",
-  "signatures" : [
-      { "keyid" : "<KEYID>",
-        "sig" : "<SIGNATURE>" }, 
-  "..."
-  ]
-}
-```
+### 4.2 General principles
 
-Where, ROLE is a dictionary whose `"_type"` field describes the metadata type (as
-described in sections 4.3 and 4.4). KEYID is the identifier of the key signing
-the ROLE dictionary. SIGNATURE is a signature of the canonical JSON form of
-ROLE.
+#### 4.2.1 Key formats
```

Comment:
I wonder if we want to mention that implementations may serialize to disk using whatever format for private keys they would like to use (I don't think this is clear, and tools like -golang and -rs opted for this to simplify the impl).

Reply:
Added a note.